.github/workflows/ci.yml

```diff
@@ -41,7 +44,7 @@ jobs:
       - name: Build and run tests
         id: gradle
         timeout-minutes: 10
-        run: ./gradlew check
+        run: ./gradlew check verifyRoborazzi
```

Comment:
This is just a personal observation, but I prefer using `verifyRoborazzi**Debug**` since the test runs twice - once for debug and once for release.

Reply:
I apologize, but since this might be a JVM module, there may not be any build types available. 🙇